src/controllers/LeaderboardChannelController.ts

```diff
@@ -9,11 +9,13 @@ export async function updateLeaderboardChannel(leaderboardChannel: TextChannel):
 
   const embeds = leaderboardContent.map((content, index) => {
     const embedCtr = leaderboardContent.length > 1 ? `(${index + 1}/${leaderboardContent.length})` : "";
+    const normIconURL = "https://raw.githubusercontent.com/mattwells19/UNCC-Six-Mans.js/main/media/norm_still.png";
```

Comment:
Was this line here on the last PR? We don't want it inside of our `.map(...)`. That unnecessarily redeclares the variable for each loop iteration. Can we move it outside of the function, please?

Reply:
Fixed in commit